### PR TITLE
add get all tests function

### PR DESCRIPTION
**Pull request by brian386**
## Notion ticket link

[Get all tests](https://www.notion.so/uwblueprintexecs/47bf4727a2444bccad4a91952898b8f3?v=9658436bf9be409e90581631a7155f16&p=b6fbc35c08384f3b901e6bbc5feda02e&pm=s)

## Implementation description
* 

## Steps to test
1.

## What should reviewers focus on?
* 

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/services/implementations/testService.ts [hunk 1]
```diff
@@ -66,6 +66,34 @@ class TestService implements ITestService {
       grade: test.grade,
     };
   }
+
+  async getAllTests(): Promise<TestResponseDTO[]> {
```
Comment (mahadzaryab1) [inline backend/services/implementations/testService.ts:115]: i don't see the method in the interface committed - did you forget to commit it?

**Review by mahadzaryab1 — COMMENTED** [review]
added a comment

File: backend/services/implementations/__tests__/testService.test.ts [hunk 2]
```diff
@@ -5,9 +5,13 @@ import {
   assertResponseMatchesExpected,
   mockAdmin,
   mockTest,
+  mockTestArray,
 } from "../../../testUtils/tests";
 import MgTest from "../../../models/test.model";
 import UserService from "../userService";
+import testModel from "../../../models/test.model";
+import SchoolService from "../schoolService";
```
Comment (carissa-tang) [inline backend/services/implementations/__tests__/testService.test.ts]:
```suggestion
```
is this import being used? if not, can remove it

**Review by carissa-tang — COMMENTED** [review]
one small comment.. rest looks good!

**Review by carissa-tang — APPROVED** [review]
lgtm! just need to fix merge conflicts

**Pull request merged**